pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go

```diff
@@ -150,8 +168,35 @@ func startDistIngestion(
 		func() time.Duration { return replanFrequency.Get(execCtx.ExecCfg().SV()) },
 	)
 
+	// TODO(adityamaru): Replace this timer-based loop once we teach requesting
+	// execution details to reach out to the Resumer and trigger the generation of
+	// job specific execution details.
+	stopGeneratingExecutionDetails := make(chan struct{})
+	generateExecutionDetails := func(ctx context.Context) error {
+		execDetailsTimer := timeutil.NewTimer()
+		execDetailsTimer.Reset(persistExecutionDetailsFrequency.Get(&execCtx.ExecCfg().Settings.SV))
```

Comment:
nit: grab the cluster setting within the loop so people can update the setting without pausing the job

Reply:
or if you want to be fancy, you could use the `SetOnChange` api. E[xample](https://github.com/cockroachdb/cockroach/blob/ef850fb9773921d19cfa220fd756623c4d91505c/pkg/spanconfig/spanconfigmanager/manager.go#L120)

Reply:
changed it to a const as dt recommended above, this is going away very soon once the remaining pieces of https://github.com/cockroachdb/cockroach/issues/109671 are implemented.